### PR TITLE
lxa_iobus: node: add HTTP API based LxaRemoteNode

### PR DESCRIPTION
**Pull request by hnez**
This PR was initially a part of #39, but is more or less independent from it.

Breaking it out should allow us to get this feature merged faster, since #39 has to wait until we have a released version of the Optick Firmware.

The LxaRemoteNode allows us to directly communicate with LXA IOBus devices that are controlled by an IOBus server from external scripts, allowing accelerated development of new devices and features.

### CODE REVIEW

File: lxa_iobus/node/remote_node.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+from lxa_iobus.canopen import SdoAbort
+
+from .base_node import LxaBaseNode
+
+logger = logging.getLogger("lxa_iobus.remote_node")
+
+
+class LxaRemoteNode(LxaBaseNode):
+    @classmethod
+    async def new(cls, base_url, node_name):
```
Comment (SmithChart) [inline lxa_iobus/node/remote_node.py:14]: This is a new external interface. I would prefer if we would add some comments on how to use this.

**Review by SmithChart — APPROVED** [review]
One small note regarding comments to the new interface. I'll leave it up to you, if you can manage to fix this in this iteration.

Otherwise LGTM.

File: lxa_iobus/node/remote_node.py (hunk 1 above)
Comment (hnez) [reply lxa_iobus/node/remote_node.py:14]: Hi, I've added a few docstrings to the `LxaRemoteNode` class now.
I've also looked into `LxaBaseNode`, but I think most of the methods there are quite self descriptive already (e.g. `ping`, `set_locator_state`, `invoke_isp`, `info`).

Comment (SmithChart) [reply lxa_iobus/node/remote_node.py:14]: Thanks for the update. For me this is now descriptive enough.

**Review by SmithChart — APPROVED** [review]

**Pull request merged**